dockerfiles/stages/1-build-deps

```diff
@@ -120,24 +123,6 @@ RUN curl --proto '=https' \
   && rm /tmp/rustup-init
 USER root
 
-# --- RocksDB Installation of a given ROCKSDB_VERSION
```

Comment:
I assume we're relying on OPAM to instruct compilation of RocksDB?

Reply:
Yes, we investigated this with @dannywillems and this version is incorrect and not event used